### DISCUSSION TITLE
bottom toolbar?

### DISCUSSION THREAD

**Discussion by intellysoft** [xdan/jodit#923] [Q&A] [answered]
good morning everyone!
How can I place the toolbar at the bottom instead of the top (in the full screen version it can stay on top)?

**Comment by xdan** [comment, marked as answer]
There's no built-in option to move the toolbar to the bottom — it renders at the top (and you'd want it on top in fullscreen anyway, like you said). The practical way is CSS: the editor is a flex column, so you can reorder the toolbar below the workplace with `order` / `flex-direction` on `.jodit-container`, and scope it to skip fullsize via the `.jodit_fullsize` class. It's a layout tweak, not a setting — happy to give the exact CSS if you want it.
